scripts/set-page-title.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: MIT
+
+"""
+This script sets the title for all translations of a page.
+It can be used to update the titles in translations.
+
+Note: Before running this script, ensure that TLDR_ROOT is set to the location
+of a clone of https://github.com/tldr-pages/tldr, and 'git' is available.
+If there is a symlink error when using the stage flag remove the `pages.en`
+directory temporarily and try executing it again.
```

Comment:
Does the script still suffer from the symlink problem? IIRC replacing of path-related `os ` functions with the newer pathlib functions fixed this problem.

Reply:
AFAIK this problem is Windows-specific. I ran "set-more-info-link.py -S" without deleting the symlink and it worked without any problems.

@kbdharun @acuteenvy do you confirm this symlink problem caused by - probably - `os` path-related functions is Windows-specific, not occuring on Unix-like platforms like macOS and Linux?

Reply:
> do you confirm this symlink problem caused by - probably - `os` path-related functions is Windows-specific, not occuring on Unix-like platforms like macOS and Linux?

`set-page-title.py --sync` and `set-more-info-link.py --sync` both work fine for me on Linux.

What exactly is this symlink problem? Like what's the error you're getting on Windows?

Reply:
> `set-page-title.py --sync` and `set-more-info-link.py --sync` both work fine for me on Linux.
> 
> What exactly is this symlink problem? Like what's the error you're getting on Windows?

As I said at the start of the message, the error is not occuring anymore, it's working fine on Windows.

I don't remember what was the error I was getting in a previous version of the script, but I had to remove the symlink to make the script work.

I'm almost sure this issue was fixed when pathlib was introduced in the code, replacing all non-pathlib path-related functions.

I just want to confirm this issue was really fixed, because the script documentation doesn't say it is Windows-specific. Maybe I will run the pre-pathlib version when I have the time.

Reply:
> As I said at the start of the message, the error is not occuring anymore, it's working fine on Windows.

Yep, can confirm the same here.